mnefun/_mnefun.py

```diff
@@ -600,7 +601,10 @@ def fetch_raw_files(p, subjects, run_indices):
                                   p.acq_ssh, finder])[0]
         remote_fnames = [x.strip() for x in stdout_.splitlines()]
         assert all(fname.startswith(p.acq_dir) for fname in remote_fnames)
-        remote_fnames = [fname[len(p.acq_dir) + 1:] for fname in remote_fnames]
+        p_dir, _ = op.split(remote_fnames[0])
```

Comment:
this assumes that the directory is only two parts like `/a/b` and not `/a/b/c` right? That's not a general/safe enough assumption